server/src/services/surveyService.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ContainerRequest } from '@azure/cosmos';
+import CosmosClient from '../databases/cosmosClient';
+import { SurveyResultRequestModel } from '../models/surveyModel';
+
+const surveyContainerName = 'Surveys';
+const containerRequest: ContainerRequest = {
+  id: surveyContainerName,
+  partitionKey: {
+    paths: ['/callId']
+  }
+};
+
+class SurveyService {
+  private cosmosClient: CosmosClient;
+
+  constructor(cosmosClient: CosmosClient) {
+    this.cosmosClient = cosmosClient;
+  }
+
+  async init(): Promise<void> {
+    await this.cosmosClient.createDatabase();
```

Comment:
I believe line 24 does not need to be invoked here. It still feels cosmosClient logic. Its not required here. Db should be instantiated as part of cosmosClient contructor.